bootstap.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash -i
+
+function install {
+	#Postgres
+	curl -o- http://apt.postgresql.org/pub/repos/apt/ACCC4CF8.asc | sudo apt-key add -
+	echo "deb http://apt.postgresql.org/pub/repos/apt/ trusty-pgdg main" | sudo tee -a /etc/apt/sources.list.d/pgdg.list
+	sudo apt-get update
+
+	#Install dependencies
+	sudo apt-get install -y curl build-essential git libcurl4-openssl-dev libreadline-dev libssl-dev libxml2-dev libxslt1-dev zlib1g-dev libpq-dev libpq5 python-psycopg2 pgdg-keyring postgresql postgresql-contrib redis-server bundler
+
+	#Install ruby
+	gpg --keyserver hkp://keys.gnupg.net --recv-keys 409B6B1796C275462A1703113804BB82D39DC0E3
+	curl -L https://get.rvm.io | bash -s stable --ruby=2.1.3
+	source /home/kieran/.rvm/scripts/rvm
+	source ~/.bashrc
+	rvm --default use 2.1.3
+
+	#Setup Postres user
+	sudo sed -i 's/md5/trust/g' /etc/postgresql/9.4/main/pg_hba.conf
+	sudo sed -i 's/md5/peer/g' /etc/postgresql/9.4/main/pg_hba.conf
+	echo "CREATE USER $USER WITH SUPERUSER LOGIN" | sudo -u postgres psql postgres
```

Comment:
You... are aware that you can totally just use `createuser`, right?  This is't MySQL